### DISCUSSION TITLE
Multiple models with the same input

### DISCUSSION THREAD

**Discussion by JimmyMa99** [facebookresearch/detectron2#4094] [Q&A] [answered]
I want to know about how to write yaml to build multiple models in one project.
Should I make the other model or  write a new meta architecture in the yaml?

**Comment by zhanghang1989** [comment, marked as answer]
New config system should solve this kind of problem

https://github.com/facebookresearch/detectron2/blob/main/configs/new_baselines/mask_rcnn_regnetx_4gf_dds_FPN_100ep_LSJ.py

**Reply by JimmyMa99** [reply]
Thanks.
But I want to train in two different backbones in one model, I don't know how to achieve that.

**Reply by JimmyMa99** [reply]
Can I register a new backbone and use both of them?
